flink-cdc-connect/flink-cdc-source-connectors/flink-connector-mysql-cdc/src/main/java/org/apache/flink/cdc/connectors/mysql/source/reader/MySqlSourceReader.java

```diff
@@ -530,7 +530,11 @@ private void logCurrentBinlogOffsets(List<MySqlSplit> splits, long checkpointId)
                 return;
             }
             BinlogOffset offset = split.asBinlogSplit().getStartingOffset();
-            LOG.info("Binlog offset on checkpoint {}: {}", checkpointId, offset);
+            LOG.info(
+                    "Binlog offset for tables {} on checkpoint {}: {}",
+                    split.asBinlogSplit().getTables(),
```

Comment:
If there are a large number of tables, it will result in logs being quite long. Would you consider truncating them?

Reply:
Yes, I will refine it. The first few tables are sufficient to identify the specific cdc source.

Reply:
This issue is resolved. Thanks for the review.